src/styles/themes/light.pcss

```diff
@@ -1,6 +1,6 @@
 .light-mode {
   --main-bg: #fff;
-  --back-bg: #fff;
+  --back-bg: #F0F1F5;
```

Comment:
are you sure you need to change light colors in src folder?

Reply:
yes